### DISCUSSION TITLE
Irreversibility issue in phase field fracture

### DISCUSSION THREAD

**Discussion by avtarsinghh1991** [idaholab/moose#20642] [Q&A Modules: General] [answered]
Hello MOOSE experts

I am using the rate-dependent phase-field fracture equation for some thermo-mechanical problems. I found that the value of the phase-field parameter decreases in certain locations. Can you please suggest any possible reasons for such phenomena?

Best
Avtar

**Comment by hugary1995** [comment, marked as answer]
It's not clear to me from your description whether you are using moose modules or your app. Can you elaborate?

**Reply by avtarsinghh1991** [reply]
I am using MOOSE module for solving the phase-field fracture.

**Reply by hugary1995** [reply]
Great, then sharing a MWE that reproduces this issue is the easiest way we can help.

**Reply by GiudGiud** [reply]
@asingh-mit any update on this?

**Reply by avtarsinghh1991** [reply]
Yes, It is solved. I used the bounds on the variable. It bounds are not used, irreversibility issue arises.
